kolibri/content/api.py

```diff
@@ -421,6 +421,17 @@ def retrieve(self, request, pk=None):
         """
         return self._cache_kolibri_studio_channel_request(identifier=pk)
 
+    @list_route(methods=['get'])
+    def kolibri_studio_status(self, request, **kwargs):
+        try:
+            resp = requests.get(get_channel_lookup_url())
+            if resp.status_code == 404:
+                raise requests.ConnectionError("Kolibri studio URL is incorrect!")
```

Comment:
does this message ever get bubbled up to the end user?

Reply:
The message isn't relevant to the end user. Might be useful for debugging, though. If the endpoint ever moves or something, then this should appear in the logs.